modules/administration-guide/partials/con_backup-and-recovery.adoc

```diff
@@ -0,0 +1,14 @@
+[id="backup-and-recovery_{context}"]
+= Backup and recovery
+
+Backing up {prod-short} involves a combination of the following processes that back up respective data:
+
+* Backing up and restoring the {prod-short} instance by using {prod-short} Operator and a xref:supported-restic-compatible-backup-servers.adoc[configured backup server]. You can back up the {prod-short} instance using custom resources or {prod-cli} to a backup server that you have configured: xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[SFTP], xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3] (or S3 API compatible storage), or xref:backups-of-che-instances-to-a-rest-backup-server.adoc[REST]. You can restore the {prod-short} instance using xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-the-checlusterrestore-object_{context}[custom resources] or xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-prod-cli_{context}[{prod-cli}].
```

Comment:
```suggestion
* Use the {prod-short} Operator and a xref:supported-restic-compatible-backup-servers.adoc[configured backup server] to back up and restore the {prod-short} instance. You can back up the {prod-short} instance using custom resources or {prod-cli} to a backup server that you have configured: xref:backups-of-che-instances-to-an-sftp-backup-server.adoc[SFTP], xref:backups-of-che-instances-to-amazon-s3.adoc[Amazon S3] (or S3 API compatible storage), or xref:backups-of-che-instances-to-a-rest-backup-server.adoc[REST]. You can restore the {prod-short} instance using xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-the-checlusterrestore-object_{context}[custom resources] or xref:restoring-a-che-instance-from-a-backup.adoc#restoring-a-che-instance-from-a-backup-using-prod-cli_{context}[{prod-cli}].
```
Edit to avoid the sentence fragment and make more active.